catalogs/qhimm.xml

```diff
@@ -1392,6 +1392,38 @@ E.g., Bolt2 = Thundara</Description>
       </Tags>
     </Mod>
 
+## Chthonic Harmony ##
+	<Mod>
+	    <ID>a22e6110-d56e-4e7e-ae4b-0eb7ebba03fd</ID>
+	    <Name>Chthonic Harmony</Name>
+	    <Author>Numerous Authors, compiled by ChthonVII</Author>
```

Comment:
Can you please update this section mentioning at least the most important ones? Ideally shall be all though, if you can even better. Thank you